MAVProxy/tools/MAVExplorer.py

```diff
@@ -1176,6 +1183,38 @@ def cmd_paramchange(args):
         vmap[pname] = pvalue
     mestate.mlog.rewind()
 
+def cmd_logmessage(args):
+    '''show log message information'''
+    mlog = mestate.mlog
+    usage = "Usage: logmessage <help|download>"
+    if len(args) > 0:
+        if args[0] == 'help':
+            if len(args) < 2:
+                print(usage)
+                return
+            if hasattr(mlog,'metadata'):
+                mlog.metadata.print_help(args[1])
+            return
+        if args[0] == 'download':
+            # download XML files for log messages
```

Comment:
Length-wise, this is on the border of making a new separate `cmd_logmessage_download` a good idea.

Reply:
Now that download is now implemented on pymavlink side, this code is shorter, so I maybe length is now OK to keep here?